### PR TITLE
Draft Xyiv blog post 

### PR DESCRIPTION
**Pull request by hpk42**

### CODE REVIEW

File: _posts/2019-05-08-xyiv.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+riding on the waves of the coming Rustocalypse. 
+
+<img src="../assets/blog/dat-logo.png" width="100" style="float: right; margin-right: 10px" /> 
+Speaking of Rust, Sid, Franz and Matze were progressing continously on their with their Rust work on [Arso](https://arso.xyz/), a decentralized archiving project based on the [DAT P2P protocols](https://datproject.org/). Several conversations revolved around potential integrations of Delta Chat Bots with these decentralized archiving efforts. Conversations are about integrating decentralized approaches rather than trying to build "replace-the-world" platform projects that try to convince others to use and base everything on one's platform.  Arguably, decentralization projects often are often overly focused on technical decentralization but produce island platforms that do not integrate with one another.  The future of decentralization maybe lies more on **integrations at the UX level** so that users can make actual use of the wealth of interesting decentralization projects -- rather than adding yet more clever technical algorithms to any single project.
```
Comment (adbenitez) [inline _posts/2019-05-08-xyiv.md]: "Arguably, decentralization projects **often are often** overly focused"
not sure about this???

Comment (adbenitez) [inline _posts/2019-05-08-xyiv.md]: "Speaking of Rust, Sid, Franz and Matze were progressing **continously** **on their with their** Rust work on"

s/continously/continuously/
and use only one of "on their" or " with their"

**Review by adbenitez — CHANGES_REQUESTED** [review]

File: _posts/2019-05-08-xyiv.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
+---
+title: Location matters! UX testing, the Rustocalypse and other Delta Xyiv happenings ...
+author: holgax
+---
+
+<img src="../assets/blog/xyiv-to-kyiv2.png" width="200" style="float: left; margin-right: 10px" /> 
+In the last couple of weeks, location mattered deeply for Delta Chat, the "Chat over E-mail"
+messenger. It does make a difference if software development is situated in a Starbucks
+in San Francisco or around a refurbished coffee-bus in Kyiv, the embattled Ukrainian city just 200km away both from a war zone and from Czernobyl. Some of you prelinarily noticed the appearance of "on demand location streaming" in the advanced Android app settings.  This unique feature was only silently released ahead of our "Xyiv" gathering which happened in the last week of April 2019.  This post provides glimpses into what happened that week -- with the foreshadows of the coming *Rustocalypse* and evolving new takes on enmeshing UX, crypto and implementation activities in the wider decentralization space.
```
Comment (r10s) [inline _posts/2019-05-08-xyiv.md]: Chernobyl

File: _posts/2019-05-08-xyiv.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+would not have been possible.*" 
+
+<img src="../assets/blog/xyiv-mate.jpg" width="200" style="float: right; margin-right: 10px" /> 
+The "X" in Xyiv can be taken to symbolize the "unknown", the twisted and winded paths
+of developments one can not predict nor expect. And the unexpectable did arrive 
+during our Kyiv gathering. Friedel played with [C2Rust](https://c2rust.com/),
+a new tool to convert C-code to Rust and, applying a few more magical spells, managed to produce a rough Rust version of Delta Chat Core. The new code even remained readable and almost compiled! 
+Several discussions and hacking-day-and-night sessions later, the Rustocalypse appears 
+to be approaching, with the [new Delta Chat Rust core](https://github.com/deltachat/deltachat-core-rust/) poised to become the new base for all Delta Chat apps. The C2Rust translation was 
+possible only because DCC is written in plain C -- by contrast, C++ code can not
```
Comment (r10s) [inline _posts/2019-05-08-xyiv.md:73]: i think it is not clear to the average reader that Delta Chat Core == DCC, maybe change the term "Delta Chat Core" a few lines above to "Delta Chat Core (DCC)" to make this clearer.

File: _posts/2019-05-08-xyiv.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+if someone threatens to break your fingers (or worse) until you provide access to your device.
+It is in those situations where e. g. location streaming becomes a more important security feature 
+than than the crypto algorithm because then your supporters can know where you are and 
+better arrange for help. 
+
+# Gaming with decentralization 
+
+<img src="../assets/blog/xyiv-cards1.png" width="400" style="float: left; margin-right: 10px" /> 
+
+The game on sunday, secretly prepared by Xenia with graphics from Demian, marked kind of a culmination of an already eventful week. Participants, including locals, were put into two groups, "Cubes" and "Spheres".  Players from both teams drew a card and played a role -- journalist, whistleblower, spy, politician with each side co-ordinated by one person sitting on Delta Chat desktop at the "basecamp" location.  Players turned on location streaming and had to carry out a complex set of concurrent htasks, with a lot of fun ensuing dynamics and confusion.  Special points were gained when one side managed to "jail" the other side's politician or journalist.
```
Comment (r10s) [inline _posts/2019-05-08-xyiv.md]: tasks

File: _posts/2019-05-08-xyiv.md [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+Several players were eager to repeat and refine the game in their home towns. Also some 
+secretive talks were overheart around *Operation Werewolf* ... let further gaming commence :) 
+
+# The party and special thanks to ... 
+
+<img src="../assets/blog/xyiv-koshey.jpg" width="200" style="float: left; margin-right: 10px" /> 
+<img src="../assets/blog/xyiv-anna2.jpg" width="200" style="float: right; margin-left: 10px" /> 
+Lastly, after the game, we had a party that was so nice that almost nobody had time
+to take pictures. There are rumors it took place in some cellar and that there was
+projected live-graphic coding at play, as well as shortages and abundances of alcohol and that some folks only managed to get back well after the sun had set .... but in any case special thanks go to Anna who was our local support for arranging and caring for most of what was described here in this post.
```
Comment (r10s) [inline _posts/2019-05-08-xyiv.md]: not totally sure, but i think you mean "some folks only managed to get back well after the sun arose". however, no idea about which folks you're talking here ;)

**Review by r10s — CHANGES_REQUESTED** [review]
apart from the few comments above: lgtm :) great post :)

**Review by r10s — APPROVED** [review]

File: _posts/2019-05-08-xyiv.md (hunk 1 above)
Comment (r10s) [reply _posts/2019-05-08-xyiv.md]: i think the second "often" is too much. will change that.

**Pull request merged**